evergreen.yml

```diff
@@ -538,10 +538,11 @@ functions:
         working_dir: mongosql-odbc-driver
         script:  |
           ${prepare_shell}
+          echo "Minor version: $MINOR_VERSION"
```

Comment:
This may be debugging code but tbh I find this kind of stuff useful in evergreen tasks. Was this intentionally left in the PR? Are there other things you had in here one point but removed (maybe `echo Major version: "$MAJOR_VERSION"`?).

Reply:
I had it for debugging purposes when I thought we were going to have to support side-by-side. This is the only thing I added because provided the major and minor version, or said "snapshot".